srcpkgs/LACT/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'LACT'
+pkgname=LACT
+version=0.5.4
+revision=1
+archs="x86_64* aarch64*"
```

Comment:
Uses an AMDGPU cargo crate which only compiles on 64bit

Don't think this matters, since using a modern amd gpu on 32bit arm seems extemely unlikely, but not working on i686 is unfortuneate.